ReactNativeClient/lib/services/KeymapService.ts

```diff
@@ -0,0 +1,202 @@
+
+	static defaultKeymap: KeymapItem[] = [
+		{ accelerator: 'CommandOrControl+N', command: 'newNoteItem', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+T', command: 'newTodoItem', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+S', command: 'synchronise', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+P', command: 'printItem', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+H', command: 'hideJoplin', platforms: ['darwin'] },
+		{ accelerator: 'CommandOrControl+Q', command: 'quit', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+,', command: 'preferences', platforms: ['darwin'] },
+		{ accelerator: 'Command+W', command: 'closeWindow', platforms: ['darwin'] },
+		{ accelerator: 'CommandOrControl+Alt+I', command: 'insertTemplate', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+C', command: 'copy', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+X', command: 'cut', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+V', command: 'paste', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+A', command: 'selectAll', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+B', command: 'textBold', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+I', command: 'textItalic', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+K', command: 'textLink', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+`', command: 'textCode', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+T', command: 'insertDateTime', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: shim.isMac() ? 'Shift+Command+F' : 'F6', command: 'focusSearch', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+F', command: 'showLocalSearch', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+S', command: 'focusSidebar', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+L', command: 'focusNoteList', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+N', command: 'focusNoteTitle', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: 'CommandOrControl+Shift+B', command: 'focusNoteBody', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
+		{ accelerator: shim.isMac() ? 'Option+Command+S' : 'F10', command: 'toggleSidebar', platforms: ['darwin', 'win32', 'linux', 'freebsd'] },
```

Comment:
Same here - the default config should be static without dynamic check on platform.